tower-defense/src/tower_selecting_hand/mod.rs

```diff
@@ -68,6 +68,7 @@ impl Component for TowerSelectingHand<'_> {
                     cards[index] = Card::new_random();
                 }
                 game_state.left_reroll_chance -= 1;
+                game_state.rerolled_count += 1;
```

Comment:
I think it's not a good idea to keep two opposite data in game_state.

Reply:
Can you suggest a better way?

Reply:
Keep only one of them. Why you need both field?

Reply:
https://github.com/NamseEnt/namseent/pull/1069/files#diff-438a654ec124f4f54287542e6146773dc7ac909ba3ed03d6f8bd7acf61a73caaL116-L118

An error occurred in the code above because the `extraroll item` does 'left_reroll_chance += 1'.

Reply:
then I guess `extraroll item` should affect to max reroll count.

Reply:
(max reroll count + rerolled count) vs (rerolled count + left reroll chance)
Ok, both are 2 field. OK